raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,131 @@
+## VCGENCMD
+
+`vcgencmd` is a command line utility that can get various pieces of information from the Videocore4 GPU on the Raspberry Pi. Much of the information available is only of use to internal Raspberry Pi developers, but there are number of very useful options available to end users that will be described here.
```

Comment:
Perhaps "internal Raspberry Pi developers" should be changed to "Raspberry Pi engineers" for consistency with the vcdbg page?

Also, "there are number of " -> "there are a number of "